templates/archive_wal.sh.j2

```diff
@@ -13,10 +13,16 @@ wal_archive_dir={{ (postgresql_backup_dir ~ '/wal_archive') | quote }}
 file_path="$1"
 file_name="$2"
 
+{% if ":" in postgresql_backup_dir %}
```

Comment:
you sure you don't want to have an additional flag for "please rsync it"? just in case someone's got a `:` in their path?

Reply:
rsync is always used even if it's a mounted path, but yeah, I could make this an overrideable option that defaults to `':' in postgresql_backup_dir`.

Reply:
In `backup.py` I check if the first character of `postgresql_backup_dir` is `/`, that is probably a more reasonable assumption (I don't think rsync would ever treat that as anything other than a local path)?

Reply:
oh a leading forward slash, yes, that's better for sure!